unrpyc.py

```diff
@@ -213,19 +213,18 @@ def parse_sl_custom_names(unparsed_arguments):
 def worker(arg_tup):
     args, filename = arg_tup
     try:
-        if args.write_translation_file:
-            return extract_translations(filename, args.language)
+        if args.tl_step == 1:
+            tl_data = extract_translations(filename, args.translate)
+            return tl_data
+        elif args.tl_step == 2:
+            tl_inst = translate.Translator(args.translate)
+            tl_inst.dialogue, tl_inst.strings = args.translate2
```

Comment:
I'd prefer just making a different worker for the first part of the translation system instead of messing with flags.

Reply:
I had the thought for this variant but wasn't aware to extent it like this is ok for you.